types/test.tsx

```diff
@@ -0,0 +1,42 @@
+/** @jsx x */
+import x = require('xastscript')
```

Comment:
Two notes:
1. I opted to create two test files, one for plain ts and another for tsx, to ensure the script can be used independent of jsx
2. this file does not test fragments, custom fragment syntax `/** @jsxFrag null */` is only supported in typescript 4 and above. Out current typings support this usage, however dtslint does not appear to offer a way to run tests in multiple versions of typescript on the same typing file. We would need to have two typings files which would be identical except the header comment.

```
// TypeScript Version: 3.7
```
vs
```
// TypeScript Version: 4.0
```

and I'd like to avoid splitting the typing just for a header.
Open to other suggestions though.